### PR TITLE
StefanSundin.Superf4 version 1.4

### PR DESCRIPTION
**Pull request by Shcookius**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you validated your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/15696)

### CODE REVIEW

File: manifests/s/StefanSundin/Superf4/1.4/StefanSundin.Superf4.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# Created using wingetcreate
+
+PackageIdentifier: StefanSundin.Superf4
+PackageVersion: 1.4
+PackageLocale: en-US
+Publisher: Stefan Sundin
+PackageName: superf4
```
Comment (ItzLevvie) [inline manifests/s/StefanSundin/Superf4/1.4/StefanSundin.Superf4.locale.en-US.yaml:7]:
```suggestion
PackageName: SuperF4
```

File: manifests/s/StefanSundin/Superf4/1.4/StefanSundin.Superf4.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,11 @@
+# Created using wingetcreate
+
+PackageIdentifier: StefanSundin.Superf4
```
Comment (ItzLevvie) [inline manifests/s/StefanSundin/Superf4/1.4/StefanSundin.Superf4.locale.en-US.yaml:3]:
```suggestion
PackageIdentifier: StefanSundin.SuperF4
```

File: manifests/s/StefanSundin/Superf4/1.4/StefanSundin.Superf4.locale.en-US.yaml [hunk 3]
```diff
@@ -0,0 +1,11 @@
+# Created using wingetcreate
+
+PackageIdentifier: StefanSundin.Superf4
+PackageVersion: 1.4
+PackageLocale: en-US
+Publisher: Stefan Sundin
+PackageName: superf4
+License: GPL v3
+ShortDescription: Hard kills programs similar to the task manager using Alt+F4. Useful for unstable programs. Not my property but has been brought here for your convienence.
```
Comment (ItzLevvie) [inline manifests/s/StefanSundin/Superf4/1.4/StefanSundin.Superf4.locale.en-US.yaml:9]:
```suggestion
ShortDescription: Kill programs instantly with Ctrl+Alt+F4.
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

**Comment by denelon** [comment]
@ItzLevvie we were rolling out a change to reduce some duplicate label requirements "Validation-Completed" and "Azure-Pipeline-Passed" are no longer both required. @OfficialEsco found one other PR that managed to get merged during this time.

**Pull request merged**